python/mxnet/gluon/block.py

```diff
@@ -1059,7 +1059,7 @@ def _call_cached_op(self, *args):
             out = [out]
         return _regroup(out, self._out_format)
 
-    def optimize_for(self, x, *args, backend=None, backend_opts=None, clear=True, **kwargs):
+    def optimize_for(self, x, *args, backend=None, clear=True, static_alloc=False, static_shape=False, **kwargs):
```

Comment:
I like this, can we do the same for hybridize and eliminate backend_opts altogether?
Change:
```
    def hybridize(self, active=True, backend=None, backend_opts=None, clear=True, **kwargs):
```
To:
```
    def hybridize(self, active=True, backend=None, clear=True, static_alloc=False, static_shape=False, **kwargs):
```

Reply:
Done